docs/translations/api-docs/data-grid/grid-list-col-def.json

```
@@ -0,0 +1,16 @@
+{
+  "interfaceDescription": "Column Definition interface used for the list view column.",
+  "propertiesDescriptions": {
+    "field": {
+      "description": "The column identifier. It&#39;s used to map with GridRowModel values."
+    },
+    "align": { "description": "Allows to align the column values in cells." },
+    "cellClassName": { "description": "Class name that will be added in cells for that column." },
```

future –> present tense (implied)

```suggestion
    "cellClassName": { "description": "Class name added to cells for that column." },
```